### DISCUSSION TITLE
Perspective support in Version 2.0

### DISCUSSION THREAD

**Discussion by tnylea** [tailwindlabs/tailwindcss#2726] [Ideas]
Hey gang!

Any chance of getting the `perspective` property in version 2.0? https://www.w3schools.com/cssref/css3_pr_perspective.asp

There are a lot of cool ways to display elements when utilizing this property. Currently when I create a new project in Tailwind and I need to use this property I will just add it as custom CSS. I could also see adding it as a plugin but just wanted to check if you guys have given any thought into adding it into the core of version 2.0.

Appreciate ya guys!

I got a Tailwind shirt last week 🙌  and I love it.

![shirt](https://user-images.githubusercontent.com/601261/98009728-bc8dfd80-1daa-11eb-8158-0362311a815b.jpeg)

Thanks for all the awesomeness!

**Comment by pjschreifels** [comment]
👍**Upvote**

`perspective` and `perspective-origin` support would be great!

**Comment by XPD-Kasun** [comment]
You can check out [tailwind 3d transform plugin](https://xpd-kasun.github.io/tailwind-3dtransform-plugin/guide/perspective). If needed I would like to integrate to tailwind repo too.
